keyboards/mode/m256ws/m256ws.c

```diff
@@ -0,0 +1,154 @@
+/* Copyright 2022 Gondolindrim
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "eeconfig.h"
```

Comment:
```suggestion
```

Reply:
Committed in [2e147df](https://github.com/qmk/qmk_firmware/pull/20502/commits/2e147dfbd76ace79d37fbf09093c2be8bfe0a7a8)